packages/plugins/geth/src/index.js

```diff
@@ -42,6 +42,7 @@ class Geth {
           if (err) {
             this.logger.error(`Error launching blockchain process: ${err.message || err}`);
           }
+          this.setupDevTxs();
```

Comment:
we can only start the regular txs once the node has started.

Reply:
This was previously prevented by using `blockchain:client:provider` which relied on `proxy:endpoint:get` which waited for `blockchain:started`.

Reply:
I Agree but in this case because we're doing that anymore, we have to ensure it's only done after it started (and it breaks if it's not here)